pages/bills/[id]/index.js

```diff
@@ -28,46 +28,60 @@ export default function BillPageDetails() {
 
   return (
     <Wrapper>
-      <StyledLink href='/'>
+      <StyledLink href='/create'>
         <Back style={{ color: "var(--clr-accent)" }} />
       </StyledLink>
       <BillDetails title={bill.title} amount={bill.amount}>
         {bill.participants.map((participant, id) => (
           <ListItem key={id}>
             {participant.name}
-            <BalanceWrapper>
-              {participant.balance}
+            <ItemWrapper>
+              {participant.balance > 0
+                ? `owes ${participant.balance} $`
+                : participant.balance < 0
+                ? `is owed ${-participant.balance} $`
+                : "has no balance"}
               <br />
```

Comment:
A `br` element should not be used like you did. [See MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns)